### ISSUE TITLE
Jenkins Host Connection Not Found

### ISSUE THREAD

**Issue by esafirm** [tabeyti/jenkins-jack#13]
When i tried to use `Jenkins Jack: Jenkins Host Connection` i got this error message

```
command 'extension.jenkins-jack.connections' not found
```

Using version 1.0.1 (VSCode ext)

**Comment by tabeyti** [comment]
Hey @esafirm ,

Are you sure you have the correct Jenkins connection information in your `settings.json`? See [here](https://github.com/tabeyti/jenkins-jack#configuration) for an example.

**Comment by eboboshka** [comment]
@tabeyti I have the same problem, too.
VSCode
```
Version: 1.40.2
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
```
Ext:
```
Name: Jenkins Jack
Id: tabeyti.jenkins-jack
Version: 1.0.1
```
settings.json
```
    "jenkins-jack.jenkins.connections": [
        {
            "name": "localhost",
            "uri": "http://localhost:8080",
            "username": "admin",
            "password": "1b..........1d",
            "active": true
        }
    ],
```

**Comment by eugenebnd** [comment]
Same here

`Version:          Code 1.42.0 (ae08d5460b5a45169385ff3fd44208f431992451, 2020-02-06T10:51:34.058Z)
OS Version:       Windows_NT x64 6.1.7601`

**Comment by shane-od** [comment]
Same here. Version 1.0.1 on Mac

**Comment by whatarule** [comment]
This worked for me.
- put "files.associations": { "Jenkinsfile": "groovy" } in settings.json

**Comment by tabeyti** [comment]
Hey guys,

There was a bug where the "Jenkins Host Connection Not Found" would appear, even after updating the connection information to something valid.

If this is your issue, it's been [fixed in 1.10](https://github.com/tabeyti/jenkins-jack/blob/master/CHANGELOG.md#fixed). 

Please let me know if this problem is still persisting.

**Comment by tabeyti** [comment]
Closing as this has been fixed in version 1.1.0
